R/pkg/R/mllib_recommendation.R

```diff
@@ -82,6 +82,9 @@ setClass("ALSModel", representation(jobj = "jobj"))
 #' statsS <- summary(modelS)
 #' }
 #' @note spark.als since 2.1.0
+#' @note the input rating dataframe to the ALS implementation must be determinate. If the training
```

Comment:
deterministic?

Reply:
Not sure deterministic or determinate which one is more correct. I use determinate just because RDD deterministic level has DETERMINATE, UNORDERED and INDETERMINATE.

Let me change to deterministic.

Reply:
Yeah despite the name of the constants, I think the word here is 'deterministic'.